### PR TITLE
update location

### PR DESCRIPTION
**Pull request by trabaldini**

### CODE REVIEW

**Comment by trabaldini** [comment]
Got the text from the website of the Monticello House. Should I add more?

**Comment by marcoow** [comment]
Let's add the text about lunch etc. from the commit I linked to as well.

**Comment by trabaldini** [comment]
Added the lunch and dinner text.

File: index.html [hunk 1]
```diff
@@ -528,16 +528,28 @@ <h3>Your instructor Mike North</h3>
     <div class="row">
       <div class="col-xs-12">
         <h3>The Venue</h3>
-        <p>The Workshop will be held in <strong>central London</strong> - we will announce the exact venue in the next few weeks.</p>
+        <div class="col-sm-3 col-xs-12">
+          <img src="images/location.jpg" alt="Image of location" class="img-circle img-responsive center-block">
+        </div>
+        <div class="col-sm-9 col-xs-12">
+          <p>The Workshop will be held in the <strong>Monticello House of Anglo Educational Services</strong>. It is located in <strong>Bloomsbury</strong>, which is considered to be the academic and cultural heart of London. <strong>Lunch will be catered at the venue</strong> and of course we'll have coffee and drinks available during the day.</p>
+          <p>Also, as it has become a tradition already, <strong>simplabs will invite all workshop attendees for dinner</strong> on the first day of the workshop 🍔🍹.</p>
```
Comment (marcoow) [inline index.html]: Sorry, not this part please

**Comment by trabaldini** [comment]
Ok, removed that part.

**Pull request merged**